modules/livewrappedBidAdapter.js

```diff
@@ -177,14 +184,33 @@ function hasPubcid(bid) {
 }
 
 function bidToAdRequest(bid) {
-  return {
+  var adRequest = {
     adUnitId: bid.params.adUnitId,
     callerAdUnitId: bid.params.adUnitName || bid.adUnitCode || bid.placementCode,
     bidId: bid.bidId,
     transactionId: bid.transactionId,
-    formats: bid.sizes.map(sizeToFormat),
+    formats: getSizes(bid).map(sizeToFormat),
     options: bid.params.options
   };
+
+  if (bid.mediaTypes && bid.mediaTypes.banner && bid.mediaTypes.native) {
+    adRequest.banner = true;
+  }
+
+  if (bid.mediaTypes && bid.mediaTypes.native) {
```

Comment:
I have a personal connection with using the `deepAccess` util for these kinds of checks.

`if(deepAccess(bid, 'mediaTypes.native')`

But definitely not necessary, just like to point it out so people are aware of it!